### DISCUSSION TITLE
Should we encourage discussions over "education" issues?

### DISCUSSION THREAD

**Discussion by binyamin** [11ty/eleventy#1452] [Uncategorized]
Currently, there's an issue template for "education". Should this be replaced with a link to the discussions?

**Comment by zachleat** [comment]
Yes! I like that there is an option to `Mark Answer` on Discussions!

**Reply by binyamin** [reply]
We should probably delete that issue template, then

**Reply by zachleat** [reply]
Yeah and maybe reuse some of the language there in the GitHub discussion one. I’d also probably move that one higher in the list if we could!

**Reply by Ryuno-Ki** [reply]
Would be great if there was a wiki option as well to aggregate the knowledge we gained from a discussion.

**Reply by binyamin** [reply]
Alternatively, all knowledge could go to the website.

**Reply by Georgegriff** [reply]
Use GitHub API to generate FAQ pages from discussion threads using 11ty data 🤯

**Reply by Ryuno-Ki** [reply]
Wait, it's already covered by an API?! Link to docs, please :-)

**Reply by Georgegriff** [reply]
Unfortunately this was more wishful/bluesky thinking of what would be pretty awesome, I wonder if someone knows a github contact to ask if this is coming to their public APIs could open up some pretty interesting options for scraping github data for sites 🤔 

Sorry if I got you too excited :(

**Reply by Ryuno-Ki** [reply]
Wait a second.

I found https://developer.github.com/v3/teams/discussions/ (or „TeamDiscussions” in the GraphQL schema).
However, teams seem to be a bit different thing: https://docs.github.com/en/free-pro-team@latest/rest/reference/teams

**Comment by Ryuno-Ki** [comment]
For the record: https://www.11ty.dev/blog/github-issues/

I assume that's happening at the moment (with issues being closed / locked / converted to discussions).
